hack/generate.sh

```diff
@@ -16,6 +16,9 @@ deepcopy-gen --input-dirs kubevirt.io/client-go/apis/snapshot/v1alpha1 \
     --bounding-dirs kubevirt.io/client-go/apis \
     --go-header-file ${KUBEVIRT_DIR}/hack/boilerplate/boilerplate.go.txt
 
+deepcopy-gen --input-dirs ./pkg/virt-launcher/virtwrap/api \
```

Comment:
Did you mean to do this? This is why `make generate` fails :)

Reply:
@qinqon

Reply:
The new deepcopy-gen tool annot use "--input-dirs ." you need to specify the package so I have move it from schema.go to the generate.sh script and pass the package name there, issue was that we remoge the "*generated*" files at generate.sh, I have just moved the deepcopy-gen after the generated file removal.